### ISSUE TITLE
allow to CC users to a bug

### ISSUE THREAD

**Issue by kotowicz** [django-helpdesk/django-helpdesk#18]
for original description of this bug, see http://code.google.com/p/jutda-helpdesk/issues/detail?id=102. 

The following is still required:

"There are ticketboxes on the TicketCCForm to allow an email address to view and/or
update a ticket; these currently do nothing (they are saved in the DB but unused).
I'll do some further changes to this soon to make those fields functional, and
probably add some custom templates for CC's rather than using the Submitter template."

**Comment by timthelion** [comment]
Was this fixed by https://github.com/django-helpdesk/django-helpdesk/pull/507 ?

**Comment by gwasser** [comment]
I don't believe so, this has to do with permissions to allow non-users to access the ticket to view or update just with their email. I don't think can fully fix until finished overhauling permissions and we fix up the public portal.

**Comment by uhurusurfa** [comment]
Please open a discussion with a detailed description of desired behaviour.
Closing this as it is very old.
